pkg/apis/config/feature_flags_test.go

```diff
@@ -287,6 +287,37 @@ func TestCheckWarnResourceVerificationMode(t *testing.T) {
 	}
 }
 
+// TestCheckAlphaOrBetaAPIFields validates CheckAlphaOrBetaAPIFields for alpha, beta, and stable context
+func TestCheckAlphaOrBetaAPIFields(t *testing.T) {
+	ctx := context.Background()
+	type testCase struct {
+		name           string
+		c              context.Context
+		expectedResult bool
+	}
+	testCases := []testCase{{
+		name:           "when enable-api-fields is set to alpha",
+		c:              config.EnableAlphaAPIFields(ctx),
+		expectedResult: true,
+	}, {
+		name:           "when enable-api-fields is set to beta",
+		c:              config.EnableBetaAPIFields(ctx),
+		expectedResult: true,
+	}, {
+		name:           "when enable-api-fields is set to stable",
+		c:              config.EnableStableAPIFields(ctx),
```

Comment:
NIT: stable is the default (and will always be), so this feels a bit redundant.

Reply:
Yup I completely agree, `stable` is a default. its just added to be consistent with the 

> /approve
> 
> Good by me!

thanks @abayer 🙏 

@vdemeester can we cancel the hold? I do not think there is anything pending here now. If there is a concern of array params in a `matrix` and `when` expressions, we had examples and doc updated for both features in PR #6092 and PR #6077. @Yongxuanzhang please confirm!